AndroidBound/src/main/java/solutions/alterego/androidbound/android/adapters/BindableRecyclerViewAdapter.java

```diff
@@ -45,10 +43,6 @@
 
     private SparseArray<Class<?>> mObjectIndex;
 
-    @Getter
-    @Setter
-    private RecyclerView.LayoutManager mLayoutManager;
```

Comment:
I have some use for this. If it is not causing problem, would it be possible to have it back?

Reply:
you would need the setting of the `LayoutManager`? sure. I'm not sure it works correctly though, because trying to set it after instatiation doesn't change the layout, at least not in normal `BindableRV`.